src/test/functional/shared/apiHelpers.ts

```diff
@@ -74,16 +110,16 @@ export const getOIDCToken = async () => {
   }
 };
 
-export const suspendUser = async (userId) => {
+export const suspendUser = async (userId,email) => {
```

Comment:
missing white space between the 2 parameters

Reply:
Fixed